packages/toolpad-app/src/server/localMode.ts

```diff
@@ -478,6 +478,7 @@ function expandFromDom<N extends appDom.AppDomNode>(
       apiVersion: API_VERSION,
       kind: 'page',
       spec: {
+        displayName: node.displayName,
```

Comment:
Same, with this under `attributes` it should be in `node.attributes.displayName`.